src/components/Stake.svelte

```diff
@@ -94,38 +91,40 @@
   }
 </script>
 
-<div class:hidden={false} class="action-card basis-1/2">
-  <p class="action-card-title">
-    Stake to Provider Id {providerId}
-  </p>
-  <div>
-    <ol class="ml-4 mt-4 list-decimal font-light">
-      <li>Ensure the control key has a FRQCY balance.</li>
-      <li>Click 'Stake'</li>
-      <li>This will require 1 signature to send the transaction.</li>
-    </ol>
-  </div>
-  <form class="mt-8">
-    <KeySelection
-      component="SelectControlKey"
-      selectLabel="Key to Stake From"
-      bind:selectedOption={selectedKey}
-      {validAccounts}
-      classOverrides="border-2 rounded-lg"
-    />
-    <div class="mt-6">
-      <label for="stakingInput">Amount to Stake in <span class="units">{token}</span></label>
-      <div class="input-container mt-2">
-        <input type="number" id="stakingInput" value="1" on:input={handleInput} />
+<Modal id="stake-to-provider" {isOpen} {close}>
+  <BlockSection title="Stake to Provider">
+    <form class="column w-[320px]">
+      <DropDownMenu
+        id="stake-using-key"
+        label="Wallet Address"
+        bind:selected={selectedAccount}
+        options={Array.from($providerAccountsStore.values())}
+        formatter={formatAccount}
+        onChange={() => {}}
+      />
+      <div class="mt-6">
+        <label for="stakingInput">Amount in <span class="units">{token}</span></label>
+        <div class="input-container mt-2">
+          <input type="number" id="stakingInput" value="1" on:input={handleInput} />
+        </div>
       </div>
+      <div class="flex w-[350px] items-end justify-between">
+        <button on:click|preventDefault={stake} class="btn-primary" aria-label="Stake">Stake</button>
+        <button class="btn-no-fill" on:click|preventDefault={close}>Cancel</button>
+      </div>
+    </form>
+    <span class="min-w-full border-b border-b-divider" />
+    <div>
+      <div class="label mb-2">Requirements</div>
+      <ol class="ml-3.5 list-decimal text-xs">
+        1. Ensure the control key has a <span class="units">{token}</span> balance.
+      </ol>
+      <ol class="ml-3.5 list-decimal text-xs">2. Click 'Stake'</ol>
+      <ol class="ml-3.5 list-decimal text-xs">3. This will require 1 signature to send the transaction.</ol>
```

Comment:
```suggestion
<ol class="ordered-list text-xs">
      <li>
        Ensure the control key has a <span class="units">{token}</span> balance.
      </li>
      <li>Click 'Stake'</li>
      <li>This will require 1 signature to send the transaction.</li>
</ol>
```

Reply:
fixed